routes/role.js

```diff
@@ -0,0 +1,22 @@
+import expressPromiseRouter from 'express-promise-router';
+
+import {role as Controller} from '../controllers';
+import { validateParam, validateBody, schema } from '../helpers/routeHelpers';
+
+const router = expressPromiseRouter();
+
+
+router.route('/')
+    .get(Controller.index)
+    .post(validateBody(schema.role.post), Controller.add);
+
+router.route('/*activate/:id')
```

Comment:
http://localhost:3000/role/wqeactivate/5d34b1d42db8401a4e810bce

This endpoint still switching status of activation

Reply:
That's true, but it seems to be impossible to add a single group in express route patterns right on the beginning of the route (sth like '/(de)?activate/:id', it always throws an error. I can add a validation if the prefix of 'activate' is 'de', if that's fine enough

Reply:
I think that unfortunately express-request-promise doesn't have support for regex